wolfcrypt/test/test.c

```diff
@@ -22540,24 +22540,30 @@ static int hpke_test_single(Hpke* hpke)
         ret = wc_HpkeGenerateKeyPair(hpke, &receiverKey, rng);
 
     /* seal */
-    if (ret == 0)
+    if (ret == 0) {
+        PRIVATE_KEY_UNLOCK();
```

Comment:
I would prefer the `PRIVATE_KEY_UNLOCK();` and `PRIVATE_KEY_LOCK();` are appropriately placed in the Hpke functions.

Reply:
hmm, I was wondering about that -- I checked and test.c had many unlock-lock wrappers in it, so I just followed that pattern, but now that I look again, the other wrappers there are around whole test handler routines, not locally around individual library calls as I did it.  what I did would obviously translate directly to the definitions of `wc_HpkeSealBase()` and `wc_HpkeOpenBase()`.

shall I post a followup PR to fix it?